src/matrixeventprocessor.ts

```diff
@@ -14,6 +14,7 @@ const MaxFileSize = 8000000;
 const MIN_NAME_LENGTH = 2;
 const MAX_NAME_LENGTH = 32;
 const DISCORD_EMOJI_REGEX = /:(\w+):/g;
+const REPLY_REGEX = /> <(@.*:.*)> (.*)\n\n(.*)/;
```

Comment:
This regex could be better, matching MXIDs rather than anything. Also it appears that the reply content is wrapped into `""`